src/PaylineWidget.tsx

```diff
@@ -95,16 +94,16 @@ const PaylineWidget: React.ComponentType<PropsType> = ({
     };
   }, []);
 
-  const payline = usePayline();
   useEffect(() => {
-    if (payline) {
-      payline.reset();
-      return () => payline.hide();
+    if (token) {
+      window.Payline?.Api.init(token, template); // https://support.payline.com/hc/fr/articles/360018691239-Comment-r%C3%A9initialiser-le-widget-payline-
```

Comment:
Maybe init is enought in a SSR scenario, but reset call is better in our case as we can reinitialize the widget on a SPA